library/src/test/java/com/datatorrent/lib/parser/XmlParserTest.java

```diff
@@ -65,6 +73,19 @@ protected void finished(Description description)
   }
 
   @Test
+  public void testOperatorSerialization()
```

Comment:
Curious to know purpose of this test. Isn't it something that applies to all operators ?